kms/client.go

```diff
@@ -615,28 +615,6 @@ func (c *Client) EditCluster(ctx context.Context, req *EditClusterRequest) error
 	return resp.Body.Close()
 }
 
-// ResealCluster re-seals the cluster's on-disk state with the active HSMs.
-// It returns an error if an active HSM fails to re-seal the cluster's root
-// encryption key. For example, when the HSM is currently not available.
-//
-// It requires SysAdmin privileges.
-//
-// The returned error is of type *HostError.
-func (c *Client) ResealCluster(ctx context.Context, req *ResealClusterRequest) error {
```

Comment:
Is it fine to not mark the API as deprecated and remove as is?

Reply:
Yes, since this is not even used yet. Server doesn't support. Technically breaking but it cannot really be used yet